howto/stream/access.md

```diff
@@ -22,17 +19,9 @@ If you are running the Anbox Cloud Appliance, use the following command:
 
     anbox-cloud-appliance gateway account create my-client
 
-**Deleting a token**
+### Using the token in your requests
```

Comment:
Maybe simplified to `Using the token` to correlate with `Creating a token` in the above and `Deleting a token` below. To me, if we want to keep it as is, `Creating a token for your request` sounds more natural.